test/lib/build.test.js

```diff
@@ -478,22 +485,6 @@ describe('Build Model', () => {
         });
     });
 
-    describe('user', () => {
```

Comment:
@ian-fox I believe it's because the user getter was removed in the buildModel [here](https://github.com/screwdriver-cd/models/pull/180/files#diff-26b681bd4dac10f25d5293c7595c435bL93)